Source/Plugins/CoherenceViewer/CumulativeTFR.cpp

```diff
@@ -31,17 +31,16 @@ CumulativeTFR::CumulativeTFR(int ng1, int ng2, int nf, int nt, int Fs, int winLe
     , stepLen       (stepLen)
     , nTimes        (nt)
     , nfft          (int(fftSec * Fs))
-    , convInput     (nfft)
-    , freqData      (nfft)
-    , convOutput    (nfft)
-    , fftPlan       (nfft, &convInput, &freqData, FFTW_FORWARD, FFTW_ESTIMATE)
-    , ifftPlan      (nfft, &freqData, &convOutput, FFTW_BACKWARD, FFTW_ESTIMATE)
+    , fftArray      (nfft)
+    , ifftArray     (nfft)
+    , fftPlan       (nfft, &fftArray, FFTW_FORWARD, FFTW_ESTIMATE)
+    , ifftPlan      (nfft, &ifftArray, FFTW_BACKWARD, FFTW_ESTIMATE)
     , alpha         (alpha)
     , pxys          (ng1 * ng2,
                     vector<vector<ComplexWeightedAccum>>(nf,
                     vector<ComplexWeightedAccum>(nt, ComplexWeightedAccum(alpha))))
     , windowLen     (winLen)
-    , waveletArray  (nf, vector<std::complex<double>>(int(fftSec * Fs))) // nfft breaks here...
```

Comment:
Oh yeah, this is one of those weird C++ things. Member variables are actually initialized in the order they're declared in the class rather than the order they appear in the constructor. So since `nfft` was declared after `waveletArray`, this was getting an uninitialized value for `nfft`. I fixed it by moving the declaration of `nfft`.